pkg/java/keytool.go

```diff
@@ -0,0 +1,46 @@
+package java
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+)
+
+const defaultTruststorePath = "lib/security/cacerts"
+const DefaultTruststorePassword = "changeit"
+
+func GetDefaultTruststorePath() string {
+	return filepath.Join(os.Getenv("JAVA_HOME"), filepath.FromSlash(defaultTruststorePath))
+}
+
+func GetMavenOpts(truststoreFile string) string {
+	return "-Djavax.net.ssl.trustStore=" + truststoreFile + " -Djavax.net.ssl.trustStorePassword=" + DefaultTruststorePassword
+}
+
+func ImportCert(runner command.ExecRunner, destTruststore, certfile string) error {
+	options := []string{
+		"-import",
+		"-noprompt",
+		"-storepass", DefaultTruststorePassword,
+		"-keystore", destTruststore,
+		"-file", certfile,
+		"-alias", filepath.Base(certfile),
+	}
+	log.Entry().Infof("Importing certificate: %s", certfile)
+	return runner.RunExecutable("keytool", options...)
+}
+
+func ImportTruststore(runner command.ExecRunner, destTruststore, srcTruststore string) error {
```

Comment:
should we move `ImportTruststore` store into `ImportCert` so that before adding a new cert to the new trust store it first copies the existing trust store into the new one and then adds the new cert. 
By this we only need to make a call to with `ImportCert` . what are your thoughts ?

Reply:
In the sonar step I call `ImportCert` in a loop which would lead to a multiple import of the default store.